python/ray/util/client/__init__.py

```diff
@@ -119,28 +119,16 @@ def _register_serializers(self):
         ray.util.serialization_addons.apply(ctx)
 
     def _check_versions(self, conn_info: Dict[str, Any], ignore_version: bool) -> None:
-        local_major_minor = f"{sys.version_info[0]}.{sys.version_info[1]}"
-        if not conn_info["python_version"].startswith(local_major_minor):
-            version_str = f"{local_major_minor}.{sys.version_info[2]}"
-            msg = (
-                "Python minor versions differ between client and server:"
-                + f" client is {version_str},"
-                + f" server is {conn_info['python_version']}"
-            )
-            if ignore_version or "RAY_IGNORE_VERSION_MISMATCH" in os.environ:
```

Comment:
Are we removing behavior for `RAY_IGNORE_VERSION_MISMATCH` intentionally?

Reply:
Added back. Thanks for spotting it!